schemas/collaborations.yml

```diff
@@ -17,7 +17,7 @@ person:
 # These are links associated with the software 
 link:
   # Must match one of the values in the regex
-  category: regex('^(website|code)$')
+  category: regex('^(website|github-org|docs)$')
```

Comment:
why `github-org` instead of just `github`?

Reply:
`github` is prettier